qa/rpc-tests/wallet_parsing_amounts.py

```diff
@@ -70,19 +70,6 @@ def run_test(self):
             print(errorString)
             assert(False)
 
-        # This fee is larger than the default fee and since amount=0
-        # it should trigger error
-        fee         = Decimal('0.1')
-        recipients  = [ {"address": myzaddr, "amount": Decimal('0.0') } ]
-        minconf     = 1
-        errorString = ''
-
-        try:
-            myopid = self.nodes[0].z_sendmany(myzaddr, recipients, minconf, fee)
-        except JSONRPCException as e:
-            errorString = e.error['message']
-        assert('Small transaction amount' in errorString)
-
```

Comment:
The test at (new) line 87 is not valid. It tests that a zero-fee tx can be added to the mempool, which it cannot after #6542. Setting `-minrelaytxfee=0` would allow that to pass, but I think it is better to just check that it fails with the default `-minrelaytxfee`.

Reply:
Yeah, seems like this should be in #6542, since that’s where the behavior changes.